### DISCUSSION TITLE
Filament Runout Sensor

### DISCUSSION THREAD

**Discussion by julianzillner** [mriscoc/Ender3V2S1#708] [Q&A] [answered]
Heyy Guys, have anyone from you the Ender 3 Professional Firmware? I Like the Firmware and I use me CR Touch too with it but yesterday I installed a filament runout sensor and allways when i want start a Print, he make the side line thats ofcourse okay and when he want start the print than he load out the Filament and he say i should check my filament before I want go on ...... Can anyone Help me? Thanks a lot ...

**Comment by CoryCharlton** [comment, marked as answer]
Have you tried toggling the filament runout sensor trigger setting from the default setting (HIGH to LOW or LOW to HIGH, I can't recall what the default is)?

Edit: If you haven't seen it already this page has more details that might help you diagnose: https://github.com/mriscoc/Ender3V2S1/wiki/Filament-Settings-Menu

**Reply by julianzillner** [reply]
Thanks a lot, now it works
